setup/gcp/installer/forseti_server_installer.py

```diff
@@ -0,0 +1,339 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti Server installer"""
+
+from __future__ import print_function
+import random
+
+from forseti_installer import ForsetiInstaller
+from util.utils import (
```

Comment:
ditto on all the importings here as mentioned above

Reply:
Updated.